modules/adglareBidAdapter.js

```diff
@@ -0,0 +1,92 @@
+'use strict';
+
+import {registerBidder} from '../src/adapters/bidderFactory';
+
+const BIDDER_CODE = 'adglare';
+
+export const spec = {
+  code: BIDDER_CODE,
+  aliases: ['adglare'],
```

Comment:
This alias isn't needed since it's the same as your bidder code

Reply:
The alias has been removed.